### PR TITLE
Ansible with Docker role.

### PR DESCRIPTION
**Pull request by Dshiv**
Built a role that installs Docker to a VM and runs it.

### CODE REVIEW

**Comment by Dshiv** [comment]
Files updated for Vagrant to be able to run the docker.yml via ansible.

File: ansible/docker.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+- hosts: all
+  remote_user: dshiver
+  sudo: yes
```
Comment (crashsystems) [inline ansible/docker.yml]: sudo works, but Ansible is planning on depricating it in the future. "become" is the new method.

File: ansible/roles/docker/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- name: "Add Docker key"
+  apt_key:
+    keyserver: "hkp://p80.pool.sks-keyservers.net:80"	
+    id: "58118E89F3A912897C070ADBF76221572C52609D" 
+
+- name: "Add docker source list"
+  copy:
+    dest: "/etc/apt/sources.list.d/docker.list"
+    owner: "root"
+    src: "roles/docker/files/docker.list"
```
Comment (crashsystems) [inline ansible/roles/docker/tasks/main.yml]: I recommend also specifying group and mode in copy and template tasks.

File: ansible/stage/foo [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+[foo]
+foo
```
Comment (crashsystems) [inline ansible/stage/foo]: Foo?

File: ansible/docker.yml (hunk 1 above)
Comment (Dshiv) [inline ansible/docker.yml]: Lol, I had actually used "become" "-b" in my when I used it.

File: ansible/stage/foo (hunk 3 above)
Comment (Dshiv) [inline ansible/stage/foo]: Well Why not? It's not for production anyway. :)

Comment (crashsystems) [inline ansible/stage/foo]: :P

**Comment by Dshiv** [comment]
I've made the changes you notified me about.

**Pull request merged**